cmd/registry.go

```diff
@@ -32,31 +32,64 @@ type Registry struct {
 	Specs  []*bundle.Spec
 }
 
+// Default registry configuration section
+var defaultLocalOpenShiftConfig = registries.Config{
+	Namespaces: []string{"openshift"},
+	Name:       "lo-openshift",
+	Type:       "local_openshift",
+	WhiteList:  []string{".*$"},
```

Comment:
I think you made the local_openshift whitelist more permissive to avoid confusing developers who didn't use "apb" in the name of their APB. I like this.